src/index.js

```diff
@@ -19,6 +19,21 @@ const {
  */
 const LND_PROTO_FILE_PATH = require.resolve('../proto/lnd-rpc.proto')
 
+/**
+ * Config properties that should be exposed directly on the engine
+ * @constant
+ * @type {Array<String>}
+ */
+const PUBLIC_CONFIG = [
+  'symbol',
```

Comment:
if we are adding `symbol` here, I would remove the line above that sets `this.symbol` (this would override it)

Reply:
@treygriffith I might opt for removing it, since we set the symbol in the constructor (and already check if its available)